### ISSUE TITLE
Errors in Ubuntu 22.04

### ISSUE THREAD

**Issue by JakobStr** [clariusdev/cast#92]
When trying to run the desktop-caster example I always get a bunch of errors.
This is the output of the unchanged files:

```
caster -a 192.168.1.1 -p 5828

starting caster...
error: Failed to create pixel buffer surface: 0X3003
error: Failed to create pixel buffer surface: 0X3003
error: Failed to glGenVertexArrays, OpenGL error: 0X502
...connected, streaming port: 1 -- check firewall settings if no image callback received
error: Failed to glGenVertexArrays, OpenGL error: 0X502
```

When trying to run our project (which worked with release 9.1.0), we also receive errors:
```
Error: Failed to initialize EGL: 0X3001
Error: Failed to choose configuration: 0X3008
```
We have already updated the cast.h, cast_export,... files.

Are there packages missing, which need to be installed on linux systems?
These errors also occur in docker-files and on newly installed linux pc. 

The errorFn also only returns these strings, so it is very hard to find the underlying problem.

**Comment by clariusk** [comment]
Yes, I've witnessed that too on a Linux system I have running, we had to do a lot of porting to native OpenGL from the Qt libraries in order to get the new SDK tools running. As far as I can tell it's related to the GPU driver (likely NVIDIA), since I don't have issues on other Linux desktops.

**Comment by jameslowman** [comment]
I had the same issue, now resolved fully in 10.1.1

Using:
Ubuntu 22.04
NVidia Driver Version: 510.85.02    
CUDA Version: 11.6 
NVidia RTX 3080 Ti

**Comment by jameslowman** [comment]
I had this same issue -- it is now solved with the update to 10.1.1
Ubuntu 22.04
NVidia 520 drivers

**Comment by clariusk** [comment]
@JakobStr  if this is still an issue, can you send over the GL error messages being shown, thanks
